scripts/python/test_amend_user_repository_access.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from unittest.mock import patch
+
+from github import Github
+from gql import Client
+from gql.transport.aiohttp import AIOHTTPTransport
+
+from scripts.python import amend_user_repository_access
+
+
+@patch.object(Github, "__new__")
+@patch.object(AIOHTTPTransport, "__new__")
+@patch.object(Client, "__new__")
+class TestAmendUserRepositoryAccess(unittest.TestCase):
+
```

Comment:
Was more effort than it was worth to add a smoke test here. Would of needed to deeply mock a response object.

Probably best to leave that until later

Reply:
Sure makes sense